api/search.js

```diff
@@ -53,14 +53,31 @@ router.post('/recipes', asyncHandler(async (request, response, _next) => {
     });
   }
 
-  body.query.bool.filter = {
-    range: {
-      total_time: {
-        gte: fromCookTime,
-        lte: toCookTime,
+  body.query.bool.filter = [
+    {
+      bool: {
+        should: [
+          {
+            range: {
+              total_time: {
+                gte: fromCookTime,
+                lte: toCookTime,
+              },
+            },
+          },
+          {
+            bool: {
```

Comment:
Maybe add a todo, or create an issue, for us to fix and set this straight once Noa can tell us if the user is intrested in times or not.

Reply:
I think Noa can overwrite this when she commits her change, for now I think it is wise to show as many recipes as possible as long as we don't have the sliders yet

Reply:
acked. I'm going to take care of this hopefully tonight